src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -99,9 +99,9 @@ int MPIDI_OFI_progress(int vni, int blocking);
          * for recursive locking in more than one lock (currently limited
          * to one due to scalar TLS counter), this lock yielding
          * operation can be avoided since we are inside a finite loop. */\
-        MPID_THREAD_CS_EXIT(VCI, MPIDI_global.vci_lock);         \
+        MPID_THREAD_CS_EXIT(VCI, MPIDI_VCI(MPIDI_VCI_FALLBACK).lock);         \
```

Comment:
Let's delay this commit. When other pieces, such as hashing, progress, vci parameters pass-down, etc all gets merged, we will need to update the "FALLBACK" to the actual "per-vci" locks. So let's delay this until then.